parser.go

```diff
@@ -315,6 +315,15 @@ func ParseMessage(line string) (*Message, error) {
 	return c, nil
 }
 
+// Param returns the i'th argument in the Message or an empty string
+// if the requested arg does not exist
+func (m *Message) Param(i int) string {
+	if i < 0 || i >= len(m.Params) {
```

Comment:
I really like the idea here. I was actually considering doing something similar when I saw your PR to sorcix/irc.

Personally, I would do it like this since it's less messing around with array indices. 
```go
if param, ok := m.Params[i]; ok {
   return param
}

return ""
```

Reply:
@belak that syntax doesn't work for slices.

Reply:
Ah, blargh. That's what I get for typing something up without testing it. Thanks for the PR!